### ISSUE TITLE
Inline dropdown not working under IE9

### ISSUE THREAD

**Issue by brafun** [mgcrea/angular-strap#1953]
Using bg-dropdown with inline sibling template under IE9 throws javascript error :

TypeError: Unable to get property 'contains' of undefined or null reference <button class="btn btn-lg btn-primary" aria-expanded="false" aria-haspopup="true" type="button" data-animation="am-flip-x" bs-dropdown="">

Easily reproducible by opening the AngularStrap homepage with IE9(or IE11 with emulation of IE9).
During the initial compile the error is thrown.

![image](https://cloud.githubusercontent.com/assets/2930729/11092057/e39686de-8888-11e5-9c2c-dca5b95fc0a0.png)

**Comment by brafun** [comment]
It would appear to be the classList implementation which searches for dropdown-menu class:
if (nextSibling.classList.contains('dropdown-menu')) {
classList is obviously not supported in IE9

**Comment by mgcrea** [comment]
You should use a shim: https://github.com/eligrey/classList.js
